test/mocks/tracing/mocks.h

```diff
@@ -81,5 +82,14 @@ class MockDriver : public Driver {
                const Tracing::Decision tracing_decision));
 };
 
+class MockHttpTracerManager : public HttpTracerManager {
+public:
+  MockHttpTracerManager();
+  ~MockHttpTracerManager() override;
```

Comment:
nit: move to cc file as recommended by gmock for compile speed

Reply:
sorry, I'm not following what is wrong here.

This is just a declaration, not a definition.

It's in line with gMock documentation at https://github.com/google/googletest/blob/master/googlemock/docs/cook_book.md#making-the-compilation-faster

What am I missing ?

Reply:
Sorry my mistake. Read too quickly.